setup.py

```diff
@@ -109,6 +108,7 @@ def run_tests(self):
                    "pytest-server-fixtures",
                    "pytest-timeout",
                    "pytest-xdist",
+                   "lz4"
```

Comment:
yes, there is a test that compares base LZ4 compression vs our compression. Its a performance test. I assume we still want this? If not I can excise it and remove LZ4. We should have it as a separate test that you'd have to manually run, not one that runs every time?